__tests__/endpoints.test.js

```diff
@@ -193,12 +193,34 @@ describe('GET /api/articles', () => {
     });
 });
 
-describe.only('GET /api/articles/:article_id/comments', () => {
+describe('GET /api/articles/:article_id/comments', () => {
     test('should return a 200 status code', () => {
         return request(app)
         .get('/api/articles/3/comments')
         .expect(200)
     });
+    test('should return an array of comments for the given article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .then((commentData) => {
+            expect(commentData.body.comments.length).toBe(11)
+        })
+    });
+    test('each comment should have properties of the following type', () => {
```

Comment:
what should we see if there are no comments for a particular article? what status code would this have?